<launch>
    <node name="input" pkg="maniros" type="test_input.py" output="screen"/>
    <node name="adapter" pkg="maniros" type="control_adapter.py" output="screen">
        <param name="robot_height" type="int" value="15" />
        <param name="robot_width" type="int" value="30" />
    </node>
    <node name="servo_controller" pkg="maniros" type="servo_motor_controller.py" >
        <param name="servo1_min" type="int" value="150" />
        <param name="servo1_max" type="int" value="700" />
    </node>
    <node name="dc_controller" pkg="maniros" type="dc_motor_controller.py" output="screen" />
</launch>